app/backend/src/couchers/models.py

```diff
@@ -638,15 +638,14 @@ class HostRequest(Base):
     # timezone aware start and end times of the request, can be compared to now()
     start_time = column_property(date_in_timezone(from_date, timezone))
```

Comment:
Do we need/use start_time?

Reply:
It's not used right now, no

Reply:
Is it worth removing? My impression is column_properties are somewhat resource intensive?